### DISCUSSION TITLE
2G Limit on upload

### DISCUSSION THREAD

**Discussion by homegrownhrbs** [microfeed/microfeed#120] [Q&A]
It seems like I am not able to upload any video files over 2Gigs, is this a limitation set by the Cloudflare API? Any workarounds? 

During upload the "Add New Item" page just sits at "Uploading 0.00%" if the video file is more than 2 Gigs.

![image](https://github.com/microfeed/microfeed/assets/92547255/acd852f8-0b5e-42b5-b5a9-b3c8c1d8e731)

**Comment by wenbinf** [comment]
@homegrownhrbs are you able to upload files smaller than than 2gb?

from r2 document [1], the max file size is 5gb

[1] https://developers.cloudflare.com/r2/reference/limits/

**Comment by homegrownhrbs** [comment]
Yes, Most other video files are able to be uploaded no problem. It's just the files over 2-3gb that seem to get stuck.
